bundler/helpers/v2/monkey_patches/definition_ruby_version_patch.rb

```diff
@@ -10,7 +10,7 @@ def source_requirements
         Gem::Specification.new("Ruby\0", requested_version)
     end
 
-    %w(2.5.3 2.6.10 2.7.7 3.0.5 3.2.1).each do |version|
+    %w(2.5.3 2.6.10 2.7.7 3.0.5 3.1.4 3.2.2).each do |version|
```

Comment:
I realize technically we only need the latest patch version of the prior releases, but here we'd actually needed the version `3.2` previously when we were on `3.1`, so I thought simplest to just treat it as "note all ruby versions"` as it keeps life simple.

I do wonder if we should be dropping the old Ruby 2 versions... but that's out of scope for this PR.